### DISCUSSION TITLE
Is lowercase mandatory for field names in collections?

### DISCUSSION THREAD

**Discussion by rangzen** [pocketbase/pocketbase#2902] [Q&A] [answered]
I have a collection with field named as "fieldNameFirst" and "fieldNameSecond".

If I use something like that, results had the right number of elements but fields struct are empty.

```go
type WP struct {
    FieldNameFirst  string
    FieldNameSecond string
}
var results []WP

query := r.builder.Select("fieldNameFirst", "fieldNameSecond").
    Distinct(true).
    From("collection")
err := query.All(&results)
```

It `len(results)` is right but each field of the struct slice will have an empty string.

If I change to "fieldnamefirst" in the collection and "Fieldnamefirst" in the struct, it is working, I have the value.

**Comment by ganigeorgiev** [comment, marked as answer]
You can specify the `db` struct tag with the exact field name, aka.:

```go
type WP struct {
    FieldNameFirst  string `db:"fieldNameFirst"`
    FieldNameSecond string `db:"fieldNameSecond"`
}
```
